platform/features/conductor-redux/src/README.md

```diff
@@ -0,0 +1,187 @@
+show me real time, but stale data. Why would a user want this? My feeling is that if the inner 
+OR outer ends are moved behind NOW in real-time mode then you drop into historical mode.
+* For the API itself, have ignored question of how it's namespaced / exposed. 
+Examples assume global namespace and availability from window object. 
+For now API implemented as standard standard Require JS AMDs. Could attach 
+to window from bundle.js. Perhaps attaching to window not best approach though...
+* Have not included validation (eg. start time < end time) or any other 
+business logic such as what happens when outer interval gets dragged 
+within range of inner interval. Focus is on teasing out the public API 
+right now. 
+* Time systems are vague right now also, I don't know how they're going 
+to work or whether any API has yet been specified.
+* Not clear on the differences between real-time and follow-time as it 
+concerns the time conductor? For now the API has an end bounds mode
+of FOLLOW which automatically tracks current time, and a start time mode 
+of RELATIVE. I can envision a real-time plot that is not in follow time mode, 
+but not sure what implication is for time conductor itself and how it 
+differs from an historical plot?
+* Should the time conductor be responsible for choosing time system / domain? Currently 
+it is.
+
+## Use Cases
+1. Historical session is loaded and needs to set bounds on time conductor
+2. User changes time of interest
+3. Plot controller listens for change to TOI
+4. Real-time object is loaded and time conductor is set up in real-time mode showing last 15 minutes of data.
+5. Switch to follow time. Set inner end to now, and outer end to now + 1 hour
+6. Plot controller updated on tick
+7. Plot Controller updated when user changes bounds
+8. Conductor controller needs to update bounds and mode on TC when user changes bounds
+
+### Additional possible use-cases
+1. Telemetry adapter wants to indicate presence of data at a particular time
+2. Time conductor controller wants to paint map of data availability.
+
+These use-cases could be features of the TimeConductor, but perhaps makes 
+sense to make knowledge of data availability the sole preserve of telemetry 
+adapters, not TimeConductor itself. Adapters will be ultimately responsible 
+for providing these data so doesn't make much sense to duplicate elsewhere.
+The TimeConductorController - which knows tick interval on scale (which 
+TimeConductor API does not) - could simply request data availability from 
+telemetry API and paint it into the Time Conductor UI
+
+## Example implementations of use cases
+### 1. Historical session is loaded and system sets time bounds on conductor
+``` javascript
+function loadSession(session) {
+    var tc = MCT.conductor;
+    tc.timeSystem(session.metadata.timeSystem);
+    
+    //Set start and end modes to fixed date
+    tc.startMode(tc.Modes.FIXED);
+    tc.endMode(tc.Modes.FIXED);
```

Comment:
The use cases involving modes here are all initiated from the time conductor in platform; we don't have cases where, say, a plugin would want to set this. Because of that, I'm inclined to remove modes from the public API, and (corollary to that) to handle them architecturally with a separate component from `MCT.conductor`. The same UI component would modify the strategy this component was using, and this component would update the TC start/end times in appropriate ways at appropriate times.

This would reduce the scope of the public API, and narrow the concerns of the conductor. It does introduce some challenges in differentiating between Use Cases 6 & 7 above; the start/end setters&ast; could take an `options` argument with a flag to distinguish these.

&ast; _There's a good chance we'll also want a `bounds(start, end, [options])`, particularly if `start`/`end` calls are sensitive to ordering; depends on our validation decision._